### PR TITLE
12 arti

### LINKED ISSUE #12: CRUD Articales
A "jorunalist"/admin/mod should be able to create, destroy, read and update articles.

### PR DESCRIPTION
**Pull request by QvistJ**
Closes #12

### CODE REVIEW

File: app/controllers/articles_controller.rb [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    @article = Article.new
+  end
+
+  def create
+    @article = current_user.articles.build(article_params)
+    if @article.save
+      flash[:success] = 'Article created!'
+      redirect_to root_url
+    else
+      flash[:danger] = 'Article failed!'
```
Comment (MSaurus) [inline app/controllers/articles_controller.rb]: Maybe change it to a better message. It's not the article that failed, is the creation of the article that failed. So something like 'Couldn't create an article' or 'Failed when creating article'

File: app/models/article.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# TODO: add desc
+# desc here
```
Comment (MSaurus) [inline app/models/article.rb]: Don't you have desc already at row 7?

Comment (QvistJ) [reply app/models/article.rb]: Its description...

File: app/controllers/articles_controller.rb (hunk 1 above)
Comment (QvistJ) [reply app/controllers/articles_controller.rb]: Fixed

File: app/controllers/articles_controller.rb [hunk 3]
```diff
@@ -22,7 +22,7 @@ def create
       flash[:success] = 'Article created!'
       redirect_to root_url
     else
-      flash[:danger] = 'Article failed!'
+      flash[:danger] = 'Failed article creation!'
```
Comment (MSaurus) [inline app/controllers/articles_controller.rb]: Good :)

**Pull request merged**